group_vars/all.yml

```diff
@@ -816,6 +816,37 @@ mageops_debug_token_http_header: "{{ mageops_debug_http_header_prefix }}-Token"
 # forward it from every app node at once or guess to which one the request goes.
 mageops_xdebug_proxy_remote_connection_to_loadbalancer: "{{ varnish_as_loadbalancer }}"
 
+# ---------------------------------------------
+# --------  MageOps QA Miss cache  --------
```

Comment:
I believe calling this feature QA isn't the base name, because this doesn't have anything qa specific and isn't exclusive to QA.

This naming could be potentially misleading, especially when used in different context (eg. performance testing, debugging, etc)

I would suggest calling this "Cache Bypass" instead.

With this change all mentions of mageops_qa should be updated accordingly